lib/node_modules/@stdlib/math/base/complex/flipsign/test/test.js

```diff
@@ -0,0 +1,263 @@
+	actual = cflipsign( out, 4.2, -5.5, -7 );
+
+	expected = {
+		'0': -4.2,
+		'1': 5.5
+	};
+
+	t.deepEqual( actual, expected, 'returns expected value' );
+	t.strictEqual( actual, out, 'returns output value' );
+
+	t.end();
+});
+
+tape( 'the function does not flip sign if provided NaN `y`', function test( t ) {
+	var expected;
+	var actual;
+	var out;
+
+	// Tested against Julia...
+	out = new Array( 2 );
+	actual = cflipsign( out, -4.2, 5.5, NaN );
+
+	expected = [ -4.2, 5.5 ];
+
+	t.deepEqual( actual, expected, 'returns expected value' );
+	t.strictEqual( actual, out, 'returns output value' );
+
+	t.end();
+});
+
+tape( 'the function does not flip sign if provided `+infinity` as `y`', function test( t ) {
+	var expected;
+	var actual;
+	var out;
+
+	// Tested against Julia...
+	out = new Array( 2 );
+	actual = cflipsign( out, -4.2, 5.5, PINF );
+
+	expected = [ -4.2, 5.5 ];
+
+	t.deepEqual( actual, expected, 'returns expected value' );
+	t.strictEqual( actual, out, 'returns output value' );
+
+	t.end();
+});
+
+tape( 'the function flips sign if provided `-infinity` as `y`', function test( t ) {
+	var expected;
+	var actual;
+	var out;
+
+	// Tested against Julia...
+	out = new Array( 2 );
+	actual = cflipsign( out, -4.2, 5.5, NINF );
+
+	expected = [ 4.2, -5.5 ];
+
+	t.deepEqual( actual, expected, 'returns expected value' );
+	t.strictEqual( actual, out, 'returns output value' );
+
+	t.end();
+});
+
+tape( 'the function returns a `NaN` if provided a `NaN`, positive `y`', function test( t ) {
+	var val = cflipsign( NaN, NaN, 2 );
+	t.strictEqual( isnan( val[ 0 ] ), true, 'returns expected value' );
+	t.strictEqual( isnan( val[ 1 ] ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns a `NaN` if provided a `NaN`, negative `y`', function test( t ) {
+	var val = cflipsign( NaN, NaN, -2 );
+	t.strictEqual( isnan( val[ 0 ] ), true, 'returns expected value' );
+	t.strictEqual( isnan( val[ 1 ] ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns a `NaN` if provided a `NaN`, NaN `y`', function test( t ) {
+	var val = cflipsign( NaN, NaN, NaN );
+	t.strictEqual( isnan( val[ 0 ] ), true, 'returns expected value' );
+	t.strictEqual( isnan( val[ 1 ] ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns `+0` if provided `+0`, positive `y`', function test( t ) {
+	var val = cflipsign( +0.0, +0.0, 4 );
+	t.strictEqual( isPositiveZero( val[ 0 ] ), true, 'returns expected value' );
+	t.strictEqual( isPositiveZero( val[ 1 ] ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns `-0` if provided `-0`, positive `y`', function test( t ) {
+	var val = cflipsign( -0.0, -0.0, 55 );
+	t.strictEqual( isNegativeZero( val[ 0 ] ), true, 'returns expected value' );
+	t.strictEqual( isNegativeZero( val[ 1 ] ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function returns `+0` if provided `-0`, negative `y`', function test( t ) {
```

Comment:
Similar change as above. Basically, the change is to make the description a more grammatically correct sentence.

Reply:
This comment also applies to all the descriptions which follow.